docs/versions/unversioned/sdk/barometer.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Barometer
+---
+
+Access the device barometer sensor to respond to changes in air pressure. Currently data is passed back as raw values. You will need to convert the units yourself. Below we've included information regarding the exact format data is returned in.
+
+| OS      | Units               | Provider                                                                                                | Description                                                                                                                         |
+| ------- | ------------------- | ------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| iOS     | kilopascal          | [`CMAltimeter`](https://developer.apple.com/documentation/coremotion/cmaltimeter)                       | Altitude events reflect the change in the current altitude, not the absolute altitude.                                              |
+| Android | _`hPa`_ or _`mbar`_ | [`Sensor.TYPE_PRESSURE`](https://developer.android.com/reference/android/hardware/Sensor#TYPE_PRESSURE) | Monitoring air pressure changes.                                                                                                    |
+| Web     | `N/A`               | `N/A`                                                                                                   | This sensor is not available on the web and cannot be accessed. An `UnavailabilityError` will be thrown if you attempt to get data. |
+|         |
+
+## Usage
+
+> This module was introduced in Expo v33
+
+Barometer can be used outside of the Expo client with the `expo-sensors` & `expo-sensors-interface` libraries.
+
+## Methods
+
+### `Barometer.isAvailableAsync()`
+
+Returns a promise which resolves into a boolean denoting the availability of the device barometer.
+
+| OS      | Availability                |
+| ------- | --------------------------- |
+| iOS     | iOS 8+                      |
+| Android | Android 2.3+ (API Level 9+) |
+| Web     | `N/A`                       |
+
+#### Returns
+
+- A promise that resolves to a `boolean` denoting the availability of the sensor.
+
+### `Barometer.addListener(listener)`
+
+Subscribe for updates to the barometer.
+
+```js
+const subscription = Barometer.addListener(({ pressure /* iOS Only */, relativeAltitude }) => {
```

Comment:
The way this is written, it looks like pressure is on iOS only. Maybe use newlines to make this less ambiguous:

```suggestion
const subscription = Barometer.addListener(({
  pressure,
  relativeAltitude, // iOS only
}) => {
```